crates/runtime/src/builder.rs

```diff
@@ -125,15 +128,24 @@ impl RuntimeBuilder {
         self
     }
 
+    pub fn with_accelerator_registry(mut self, accelerator_registry: AcceleratorRegistry) -> Self {
```

Comment:
Why is this needed? I would expect this to be an internal concern of the Runtime, not something people would construct externally and pass in.